src/test/core.ts

```diff
@@ -323,7 +335,7 @@ describe('snabbdom', function() {
         assert.strictEqual(elm.childNodes.length, 1);
         assert.strictEqual(elm.childNodes[0].nodeType, 3);
         assert.strictEqual(elm.childNodes[0].wholeText, 'Foobar');
-        assert.strictEqual(typeof elm.childNodes[0].testProperty, 'function');
+        assert.strictEqual(elm.childNodes[0].testProperty, reference);
```

Comment:
And why is this a `typeof` check and not a reference check, please? Perhaps no reason?